test/BaselineOutput/Common/EntryPoints/ensemble-summary.txt

```diff
@@ -20,16 +20,16 @@ AIC:                       	118.2943
 
 Coefficients statistics:
 Coefficient    	Estimate	Std. Error	z value  	Pr(>|z|)
-(Bias)         	-5.120674 	0.6998186 	-7.317145 	0 ***
-Features.thickness	2.353567  	0.4267568 	5.515007  	5.960464E-08 ***
-Features.bare_nuclei	2.435889  	0.451504  	5.395056  	5.960464E-08 ***
-Features.uniform_shape	1.944249  	0.4137097 	4.699549  	2.622604E-06 ***
-Features.uniform_size	1.786538  	0.4204066 	4.249547  	2.145767E-05 ***
-Features.bland_chromatin	1.611417  	0.4605175 	3.499144  	0.0004668236 ***
-Features.adhesion	1.38072   	0.4821554 	2.863641  	0.00418818 **
-Features.normal_nucleoli	1.345759  	0.4784136 	2.812962  	0.00490886 **
-Features.epit_size	1.083109  	0.4566914 	2.371643  	0.0177092 *
-Cat.1          	-0.7715381	0.3420695 	-2.255501 	0.02410179 *
+-5.120674 	0.6998186 	-7.317145 	0 ***
```

Comment:
Did we lose the slot names here?

Reply:
yes as suggested here.. 

after seeing the summary files change, i think we should get the back. @tfinley@gmail.com 
[https://github.com/dotnet/machinelearning/pull/2048#discussion_r245563884](https://github.com/dotnet/machinelearning/pull/2048#discussion_r245563884)


---
In reply to: [264872767](https://github.com/dotnet/machinelearning/pull/2048#discussion_r264872767) [](ancestors = 264872767)

Reply:
See other comment.

---
In reply to: [264903719](https://github.com/dotnet/machinelearning/pull/2048#discussion_r264903719) [](ancestors = 264903719,264872767)